easybuild/easyblocks/b/bazel.py

```diff
@@ -112,13 +124,47 @@ def configure_step(self):
         # See https://github.com/bazelbuild/bazel/issues/10377
         bazel_args += ' --host_javabase=@local_jdk//:jdk'
 
+        # Link C++ libs statically, see https://github.com/bazelbuild/bazel/issues/4137
+        env.setvar('BAZEL_LINKOPTS', '-static-libstdc++:-static-libgcc')
+        env.setvar('BAZEL_LINKLIBS', '-l%:libstdc++.a')
```

Comment:
I'm in favor of making this opt-in, as is done in #2272, just to prevent that we break old Bazel easyconfigs...

Any reason not to make it opt-in?

Reply:
In our setting (i.e. building Bazel with a custom toolchain) it is always (99% sure) wrong to **not** build it statically. The earlier installations/ECs of Bazel worked was mainly "by accident"/coincidence: Most likely the system libstdc++ was similar enough to the toolchain so the problem didn't lead to a failure in most cases.

Basically the situation is:
- Bazel is build using custom libstdc++ only found by LD_LIBRARY_PATH
- Bazel uses a process-wrapper binary which may be run in environments where LD_LIBRARY_PATH is not set and in some cases this can't be changed

So to me this makes this a bugfix and should be applied to all Bazel installations.

FWIW: I tested some random Bazel ECs using very old and new ones and didn't find any problem. I also don't expect any problem: It sets environment variables which could be ignored if unsupported and the linker options set should be understood by all compiler versions used.

Reply:
I stand corrected. `Bazel-0.25.2-GCCcore-8.2.0.eb` fails to build with this due ti `LIBRARY_PATH` not being set in some cases -.-

Reply:
Added a flag which makes it default to true on Bazel >= 1.0

Reply:
:+1: